internal/backend/db/dbgorm/projects.go

```diff
@@ -41,24 +81,61 @@ func (db *gormdb) UpdateProject(ctx context.Context, p sdktypes.Project) error {
 	return nil
 }
 
+func (db *gormdb) getProject(ctx context.Context, projectID string) (*scheme.Project, error) {
+	return getOne(db.db, ctx, scheme.Project{}, "project_id = ?", projectID)
+}
+
+func (db *gormdb) getProjectByName(ctx context.Context, projectName string) (*scheme.Project, error) {
+	return getOne(db.db, ctx, scheme.Project{}, "name = ?", projectName)
+}
+
+func schemaToSDKProject(p *scheme.Project, err error) (sdktypes.Project, error) {
```

Comment:
we have parseProject in the scheme file by convention 
any reason it is here instead of there ?

Reply:
it's still there. Just exctracted common logic from getByID and getByName